module/TaskManagement/src/TaskManagement/Task.php

```diff
@@ -381,6 +415,14 @@ public function getMemberRole($user){
 		return null;
 	}
 	
+	public function getOwner(){	
+		foreach ($this->members as $key=>$member){
+			if($member['role']==self::ROLE_OWNER)
+				return $key;
```

Comment:
Ok, ora mi è chiaro il motivo: perché in realtà $member non è un'istanza di User, giusto? Non cambiarlo